src/syft/lib/misc/__init__.py

```diff
@@ -51,7 +51,13 @@ def solve_ast_type_functions(path: str) -> KeysView:
     def solve_real_type_functions(path: str) -> Set[str]:
         parts = path.split(".")
         klass_name = parts[-1]
-        return set(dir(getattr(sys.modules[".".join(parts[:-1])], klass_name)))
+        # A way to walkaround the problem we can't `import torch.return_types` and
+        # get it from `sys.modules`. TODO: a general solution to this.
```

Comment:
Can we create an issue in GitHub and then put the URL to that issue here where the TODO is so that we won't forget?

Reply:
sure